src/components/RadioButton/RadioButton.md

```diff
@@ -0,0 +1,10 @@
+```jsx
```

Comment:
It could be helpful to add some light context here describing why we have RadioButtons, RadioButtons2 and now, RadioButton. maybe a word on how to choose between them?